modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/partials/minimum.jsp

```diff
@@ -0,0 +1,26 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+FDSSampleDisplayContext fdsSampleDisplayContext = (FDSSampleDisplayContext)request.getAttribute(FDSSampleWebKeys.FDS_SAMPLE_DISPLAY_CONTEXT);
+%>
+
+<frontend-data-set:headless-display
+	apiURL="<%= fdsSampleDisplayContext.getAPIURL() %>"
+	id="<%= FDSSampleFDSNames.MINIMUM_FDS_SAMPLE %>"
+/>
```

Comment:
I liked this idea :)

I think we can emphasize in a text that this page will render a FDS with only required properties

Reply:
I think that tab name is enough. There is only one FDS per tab; if there were several, title per FDS would be useful. A more verbose description in UI might make sense, but that's a slippery slope. If we start with that, we should consistently add description in UI of all tabs. For 'customized' tag, that would have something like 10 lines, cluttering up the page.

If we want a detailed explanation of sample module, what we can do is open a document for this in confluence.